smac/smbo/local_search.py

```diff
@@ -71,10 +71,9 @@ def maximize(self, start_point, *args):
         """
         incumbent = start_point
         # Compute the acquisition value of the incumbent
-        incumbent_ = impute_inactive_values(incumbent)
-        acq_val_incumbent = self.acquisition_function(
-                                            incumbent_.get_array(),
-                                            *args)
+        incumbent_ = incumbent.get_array()
```

Comment:
Distinguishing between two variables with only "_" can cause a lot of confusion or needs at least a comment in the code.